src/include/OpenImageIO/export.h

```diff
@@ -79,6 +79,7 @@
     #define OIIO_EXPORT
   #endif
   #define OIIO_LOCAL
+  #define OIIO_INLINE  __forceinline
```

Comment:
In platform.h, there's already a macro for OIIO_FORCEINLINE.

If you don't absolutely need the strict "force", then it's ok to just use the standard "inline" keyword. I don't think there are any C++11 compliant compilers that don't accept it.